src/routes/SearchResult.js

```diff
@@ -4,9 +4,253 @@ import styles from '../views/SearchResult.module.css';
+                            <link to='/'>Rooptop</link> <br/>
+                            <link to='/'>Poolside</link> <br/>
+                    
+                        </span>
+
+
+                        <span className={styles.orange}> 
+                            <br/><br/> Location <br/> <br/>
+                        </span>
+                        <span className={styles.Ash}>
+                            <link className={styles.fliterLink} to='/'>Uttara</link> <br/>
+                            <link to='/'>Banani</link> <br/>
+                            <link to='/'>Dhanmondi</link> <br/>
+                            <link to='/'>Mohammadpur</link> <br/>
+                            <link to='/'>Mirpur</link> <br/>
+                            <link to='/'>Khilgaon</link> <br/>
+                        </span>
+
+                    </div>
+                </section>
+
+            //Restaurant Block	
+                <div className={styles.Restaurants}>
+
+                    <div className={styles.resBlock}>
+                        <div className={styles.left}>
+                            <img className={styles.resImg} src={require('/images/css/Takeout.jpg')}/>
+                        </div>
+                        <div className={styles.right}>
+                            <div className={styles.large}>Takeout</div>
+                            <div className={styles.small}>17, Level - 3, Sonargaon Janapath, Rd No 3<br/>
+                            Uttara, Dhaka.<br/><br/></div>
+                            <div className={styles.medium}> Fast food<br/> 
+                            $$$<br/></div>
+                            <div className={styles.orange}>Open now</div>
+                            
+
+
+                        </div>
+                        <span className={styles.numbers}>
+                                <div className={styles.rating}>
+                                4.8
+                                </div>
+                                <div className={styles.reviewCheckins}>
+                                    778 Reviews<br/>
+                                    2458 Check-ins
+                                </div>
+                                <div className={styles.icons}>
+                                    <link to='/'><i className={styles.icon} className='fa fa-phone'></i></link>
+                                    <link to='/'><i className={styles.icon} className='fa fa-envelope'></i></link>
+                                    <link to='/'><i className={styles.icon} className='fa fa-search'></i></link>
+
+                                </div>
+                        </span>
+                            
+                    </div>
+
+                    <div className={styles.resBlock}>
+                        <div className={styles.left}>
+                            <img className={styles.resImg} src={require('/images/css/Takeout.jpg')}/>
```

Comment:
Why use require? It causes error. It should be simple like src='/images/css/Takeout.jpg'. change it everywhere